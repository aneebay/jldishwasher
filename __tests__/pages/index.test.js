import { shallow } from "enzyme";
import React from "react";
import renderer from "react-test-renderer";

import ProductGrid from "../../pages/index";

describe("With Enzyme", () => {
  it('ProductGrid shows expected title and heading text', () => {
    const app = shallow(<ProductGrid />);

    expect(app.find("title").text()).toEqual("Dishwashers");
    expect(app.find("h1").text()).toEqual("Dishwashers");
  });
});

describe("With Snapshot Testing", () => {
  it('ProductGrid shows "Create Next App"', () => {
    const component = renderer.create(<ProductGrid />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});