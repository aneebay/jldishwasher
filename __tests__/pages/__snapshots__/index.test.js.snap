// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing ProductGrid shows "Create Next App" 1`] = `
<div>
  <main>
    <h1>
      Dishwashers
    </h1>
    <section />
  </main>
</div>
`;
